README.md

```diff
@@ -34,6 +38,15 @@ This action loads and executes a private Action.  This allows private actions to
         repo-name: some-org/super-secret-action
 ```
 
+## Example usage w/ exact SHA
+``` yaml
+    - uses: invisionapp/private-action-loader@v1
+      with:
+        repo-token: ${{ secrets.REPO_TOKEN }}
+        repo-name: some-org/super-secret-action
+        repo-sha: b8a83a0
```

Comment:
It would be awesome if we could combine these into one `owner/repo@ref` like the actual `uses` field. We can do a `git rev-parse <ref>` to get an actual sha for it. Do you think that's possible?

Reply:
Removed `repo-sha` input and now support `repo-name` in the format of `{org}/{repo}@{short or long SHA}` OR `{org}/{repo}`

Reply:
Nice, thank will be handy. Sorry to keep poking at this, but can we also support refs like `master` or `v1`?

Reply:
Updated to support all refs.  SHA, branch, and tag can all be specified now.  Added additional examples to README for these as well.